scripts/zones/Quicksand_Caves/DefaultActions.lua

```diff
@@ -1,6 +1,7 @@
 local ID = zones[xi.zone.QUICKSAND_CAVES]
 
 return {
+    ['qmAMK']             = { messageSpecial = ID.text.NOTHING_OUT_OF_ORDINARY },
```

Comment:
Looking at all the other qm's we have in the codebase, they're all in `snake_case`, can you make this match? `qm_amk`, etc.